calendar_event_jitsi/data/mail_template_data.xml

```diff
@@ -0,0 +1,249 @@
+                        <a href="/calendar/meeting/decline?db=${'dbname' in ctx and ctx['dbname'] or '' }&amp;token=${object.access_token}&amp;action=${'action_id' in ctx and ctx['action_id'] or ''}&amp;id=${object.event_id.id}"
+                            style="padding: 5px 10px; color: #FFFFFF; text-decoration: none; background-color: #875A7B; border: 1px solid #875A7B; border-radius: 3px">
+                            Decline</a>
+                        <a href="/calendar/meeting/view?db=${'dbname' in ctx and ctx['dbname'] or ''}&amp;token=${object.access_token}&amp;action=${'action_id' in ctx and ctx['action_id'] or ''}&amp;id=${object.event_id.id}"
+                            style="padding: 5px 10px; color: #FFFFFF; text-decoration: none; background-color: #875A7B; border: 1px solid #875A7B; border-radius: 3px">
+                            View</a>
+                    </div>
+                    <table border="0" cellpadding="0" cellspacing="0"><tr>
+                        <td width="130px;">
+                            <div style="border-top-left-radius: 3px; border-top-right-radius: 3px; font-size: 12px; border-collapse: separate; text-align: center; font-weight: bold; color: #ffffff; min-height: 18px; background-color: #875A7B; border: 1px solid #875A7B;">
+                                ${object.event_id.get_interval('dayname', tz=object.partner_id.tz if not object.event_id.allday else None)}
+                            </div>
+                            <div style="font-size: 48px; min-height: auto; font-weight: bold; text-align: center; color: #5F5F5F; background-color: #F8F8F8; border: 1px solid #875A7B;">
+                                ${object.event_id.get_interval('day', tz=object.partner_id.tz if not object.event_id.allday else None)}
+                            </div>
+                            <div style='font-size: 12px; text-align: center; font-weight: bold; color: #ffffff; background-color: #875A7B;'>
+                                ${object.event_id.get_interval('month', tz=object.partner_id.tz if not object.event_id.allday else None)}
+                            </div>
+                            <div style="border-collapse: separate; color: #5F5F5F; text-align: center; font-size: 12px; border-bottom-right-radius: 3px; font-weight: bold; border: 1px solid #875A7B; border-bottom-left-radius: 3px;">
+                                ${not object.event_id.allday and object.event_id.get_interval('time', tz=object.partner_id.tz) or ''}
+                            </div>
+                        </td>
+                        <td width="20px;"/>
+                        <td style="padding-top: 5px;">
+                            <p><strong>Details of the event</strong></p>
+                            <ul>
+                                % if object.event_id.jitsi_meet_url:
+                                    <li>
+                                        Jitsi Meet URL: ${object.event_id.jitsi_meet_url}
```

Comment:
Please put this in an anchor element, so that the user can click on it:

```suggestion
                                        Jitsi Meet URL: <a href="${object.event_id.jitsi_meet_url}">${object.event_id.jitsi_meet_url}</a>
```

do the same for others

Reply:
@ivantodorovich .. added href tag as per your suggestion in all 3 mail templates.